src/cosmoscout/Application.cpp

```diff
@@ -738,30 +747,34 @@ void Application::closePlugin(std::string const& name) {
 void Application::connectSlots() {
 
   // Update mouse pointer coordinate display in the user interface.
-  mInputManager->pHoveredObject.connect([this](cs::core::InputManager::Intersection intersection) {
-    if (intersection.mObject) {
-      auto body = std::dynamic_pointer_cast<cs::scene::CelestialBody>(intersection.mObject);
-
-      if (body) {
-        auto radii = body->getRadii();
-        auto polar = cs::utils::convert::toLngLatHeight(intersection.mPosition, radii[0], radii[0]);
-        auto lngLat = cs::utils::convert::toDegrees(polar.xy());
-
-        if (!std::isnan(lngLat.x) && !std::isnan(lngLat.y) && !std::isnan(polar.z)) {
-          mGuiManager->getGui()->executeJavascript(
-              fmt::format("CosmoScout.state.pointerPosition = [{}, {}, {}];", lngLat.x, lngLat.y,
-                  polar.z / mGraphicsEngine->pHeightScale.get()));
-          return;
+  mInputManager->pHoveredObject.connect(
+      [this](const cs::core::InputManager::Intersection& intersection) {
```

Comment:
We usually use `Type const& name` in our code base. I know that `const Type& name` is more commonly used, but for the sake of consistency we should stick to `Type const& name`. Changing this would mean changing every source file. If we want to do this, this should be done in a separate PR.

Reply:
Yes, we should stick to east-const. This looks like the result of `clang-tidy` with the `-fix` option. Maybe we should disable it completely and leave the fixing to the developer?